packages/siopv2-oid4vp-rp-auth/src/functions.ts

```diff
@@ -98,18 +103,22 @@ export async function createRPBuilder(args: {
   }
 
   const resolution = await context.agent.identifierManagedGet(didOpts.idOpts)
+  const resolver =
```

Comment:
Where is this resolver passed to the builder like it did in the past?

Reply:
the `resolver` param is not expected in the builder any more. I'm just passing it to the verification callback function